package.json

```diff
@@ -42,12 +44,10 @@
     "@typescript-eslint/parser": "^4.16.1",
```

Comment:
I believe this is now a peer dependency which requires it to be `^5.0.0` (latest `^5.6.0`).

Reply:
Also, remove `@typescript-eslint/eslint-plugin` since it's not needed (included in `@bigcommerce/eslint-config`)